cev_eris.dme

```diff
@@ -2820,6 +2820,7 @@
 #include "zzzz_modular_occulus\code\game\objects\items\plush.dm"
 #include "zzzz_modular_occulus\code\game\objects\items\devices\magnetic_lock.dm"
 #include "zzzz_modular_occulus\code\game\objects\items\devices\traitor_devices.dm"
+#include "zzzz_modular_occulus\code\game\objects\items\tools\_tools.dm"

```

Comment:
Wait wha, what does this have to do with anything

Reply:
Welding limbs is in there.

Reply:
I....don't even know where that came from. wtf is that?